modules/tensor_mechanics/src/materials/ComputeBlockRotatedElasticityTensor.C

```diff
@@ -44,9 +44,9 @@ ComputeBlockRotatedElasticityTensor::computeQpElasticityTensor()
 {
   EulerAngles angles;
   auto grain_id = _current_elem->subdomain_id();
```

Comment:
You should probably explicitly type this to signed integer (`int`), otherwise the < 0 check won't make sense.